Feature/add admin get batch 1

corith opened the pull request.

corith committed:
Added an endpoint for admins to get batch information by id

corith committed:
Added ability for an admin to retrieve all the batches and their
information

corith committed:
Added ability to get full batch information for a client by providing
the client's email

corith committed:
Merge branch 'development' into feature/add-admin-get-batch-1

VincentJ711 reviewed (changes requested):
you need to fetch the grades for each batch using the GradeClient. make sure you null check the List<Grade> returned by the client.

corith committed:
Fixed build issues resulting from missing Feign configuration in
AdminControllerTest.java

corith committed:
Changed the way the Batches were obtained to include grades for the
associates. Added a null check to confirm that the grades are not null.
Switched from using TrainingClient to ClientService and removed Feign
auto config from AdminControllerTest.java

corith committed:
Removed unused imports in AdminController
